mkdocs/docs/contributing/newk8sstack.md

```diff
@@ -127,5 +127,8 @@ cd iter8-<your stack>
 
     You can also refer to the [KFServing](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-kfserving), [Knative](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-knative), and [Istio](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks/iter8-istio) examples.
 
-## Step 4: Submit PR
+## Step 4: Update RBAC rules
```

Comment:
I notice we start numbering with "0". This is different from our tutorials. Perhaps start at "1"